sketch_map_tool/routes.py

```diff
@@ -147,6 +148,7 @@ def create_results_post(lang="en") -> Response:
     # Request parameters
     bbox_raw = json.loads(request.form["bbox"])
     bbox = Bbox(*bbox_raw)
+    bbox_wgs84 = Bbox(*json.loads(request.form["bboxWGS84"]))
```

Comment:
```suggestion
    bbox_wgs84_raw = json.loads(request.form["bboxWGS84"])
    bbox_wgs84 = Bbox(bbox_wgs84_raw)
```

Or change the two lines above to a oneliner as well to make both look the same.

Reply:
Why would we store an additional variable here?

Reply:
alternate solution: f07ca9b